english/alchemy/alchemy.md

```diff
@@ -37,3 +37,8 @@ You slip the ring over your finger and ...
 [Congratulations!](../marshmallow.md)
 
 [Transform!](../transform/transform.md)
+
+
+After a few transformations, each more random than the next, you find yourself sitting
+in the dirt in front of an octogonal building.   Two ancient porphory pilars frame
+the sides of a large doorway.  The [doors] (http://tarainperugia.blogspot.com/2011/06/lorenzo-ghiberti-and-baptistery-doors.html)   are a magnificent. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.